ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -203,20 +273,20 @@ public void SetBorderWidth(BorderedCanvas view, int index, double width)
         {
             var border = GetOrCreateBorder(view);
             border.SetBorderWidth(ViewProps.BorderSpacingTypes[index], width);
-        }
 
-        /// <summary>
-        /// Sets whether the view is collapsible.
-        /// </summary>
-        /// <param name="view">The view instance.</param>
-        /// <param name="collapsible">The flag.</param>
-        [ReactProp(ViewProps.Collapsible)]
-        public void SetCollapsible(BorderedCanvas view, bool collapsible)
-        {
-            // no-op: it's here only so that "collapsable" property is exported to JS. The value is actually
-            // handled in NativeViewHierarchyOptimizer
+            if (border.BorderBrush == null)
+            {
```

Comment:
Seems like we should only need to have this check in case `GetOrCreateBorder` results in creating a new Border.